modules/data.atmosphere/R/download.FACE.R

```diff
@@ -33,7 +33,8 @@ download.FACE <- function(sitename, outfolder, start_date, end_date, overwrite =
   url <- paste0("ftp://cdiac.ornl.gov/.private/eCO2_Modelling/Site_Data/", site, "/", raw.file)
   print(url)
   system(paste("wget -c ", url, " -O ", out.file))
```

Comment:
@robkooper Actually, I did do that but then removed because I was concerned about messing up @bcow function.....but I do think that we could replace as:

PEcAn.utils::download.file(url, out.file)